### ISSUE TITLE
IPT - Request to Jackett Service Failed

### ISSUE THREAD

**Issue by ryanmcgr** [Jackett/Jackett#1855]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue. Duplicating issues makes it more difficult for everyone to follow. Your issue may have already been solved in the past as well.

Provide a description of the feature request or bug, the more details the better.
If you are experiencing an issue with a tracker, a **full log must be included**. Instructions for obtaining logs are here: https://github.com/Jackett/Jackett#troubleshooting

Jackett version:  0.8.199.0
Mono version (if not using Windows): 5.4.0.56 

-------------------------------
Hi all,
I've seen quite a few posts pertaining to people having issues with adding IPT or other indexers, but I've not been able to resolve my issue through upgrading or with any of the workarounds mentioned elsewhere.

As stated in the title, when I attempt to add IPT as an indexer, I only receive a "Request to Jackett Service Failed" error message. Enhanced logs are as follow:

09-18 22:29:05 Debug IWebClient.GetString(Url:https://iptorrents.com/login.php) 
**_09-18 22:29:05 Error ReflectedHttpActionDescriptor ExecuteAsync  
System.Web.Http.Action: ReflectedHttpActionDescriptor ExecuteAsync 
09-18 22:29:05 Error ApiControllerActionInvoker InvokeActionAsync  
System.Web.Http.Action: ApiControllerActionInvoker InvokeActionAsync_** 
09-18 22:29:05 Info RequiresIndexerAttribute OnActionExecutedAsync Action filter for 'Config()' 
System.Web.Http.Filters: RequiresIndexerAttribute OnActionExecutedAsync Action filter for 'Config()'
09-18 22:29:05 Info RequiresIndexerAttribute OnActionExecutedAsync  
System.Web.Http.Filters: RequiresIndexerAttribute OnActionExecutedAsync 
09-18 22:29:05 Info JackettAPINoCacheAttribute OnActionExecutedAsync Action filter for 'Config()' 
System.Web.Http.Filters: JackettAPINoCacheAttribute OnActionExecutedAsync Action filter for 'Config()'
09-18 22:29:05 Info JackettAPINoCacheAttribute OnActionExecutedAsync  
System.Web.Http.Filters: JackettAPINoCacheAttribute OnActionExecutedAsync 
09-18 22:29:05 Info ApiExceptionHandler ExecuteExceptionFilterAsync  
System.Web.Http.Filters: ApiExceptionHandler ExecuteExceptionFilterAsync 
09-18 22:29:05 Info ApiExceptionHandler ExecuteExceptionFilterAsync  
System.Web.Http.Filters: ApiExceptionHandler ExecuteExceptionFilterAsync 
09-18 22:29:05 Info IndexerApiController ExecuteAsync  
System.Web.Http.Controllers: IndexerApiController ExecuteAsync 
09-18 22:29:05 Info   Content-type='application/json; charset=utf-8', content-length=27602 
System.Web.Http.Request:   Content-type='application/json; charset=utf-8', content-length=27602
09-18 22:29:05 Info IndexerApiController Dispose  
System.Web.Http.Controllers: IndexerApiController Dispose 
09-18 22:29:05 Info IndexerApiController Dispose  
System.Web.Http.Controllers: IndexerApiController Dispose

For what it's worth, I also have an nginx reverse proxy setup, so I've changed the base path override to /username/jackett. Otherwise, I haven't fiddled with anything. Can get to IPT with no issues.

Thanks,
Ryan

**Comment by ryanmcgr** [comment]
It's probably also worth noting that I'm hosting this on a feral slot. I don't believe IPT is blocked wherever the server is hosted.. I can ping and nslookup while I'm SSH'd into the box with no issue.

Thnx,
Ryan

**Comment by kaso17** [comment]
Try running e.g. ``curl -v https://iptorrents.com/login.php` from your server please.

**Comment by ryanmcgr** [comment]
Output -

*   Trying 104.23.202.30...
* TCP_NODELAY set
* Connected to iptorrents.com (104.23.202.30) port 443 (#0)
* Cipher selection: ALL:!EXPORT:!EXPORT40:!EXPORT56:!aNULL:!LOW:!RC4:@STRENGTH
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/certs/ca-certificates.crt
  CApath: none
* TLSv1.2 (OUT), TLS handshake, Client hello (1):
* TLSv1.2 (IN), TLS handshake, Server hello (2):
* NPN, negotiated HTTP1.1
* TLSv1.2 (IN), TLS handshake, Certificate (11):
* TLSv1.2 (IN), TLS handshake, Server key exchange (12):
* TLSv1.2 (IN), TLS handshake, Server finished (14):
* TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
* TLSv1.2 (OUT), TLS change cipher, Client hello (1):
* TLSv1.2 (OUT), TLS handshake, Unknown (67):
* TLSv1.2 (OUT), TLS handshake, Finished (20):
* TLSv1.2 (IN), TLS change cipher, Client hello (1):
* TLSv1.2 (IN), TLS handshake, Finished (20):
* SSL connection using TLSv1.2 / ECDHE-ECDSA-AES128-GCM-SHA256
* Server certificate:
*  subject: OU=Domain Control Validated; OU=PositiveSSL Multi-Domain; CN=ssl411507.cloudflaressl.com
*  start date: Aug 18 00:00:00 2017 GMT
*  expire date: Feb 24 23:59:59 2018 GMT
*  subjectAltName: host "iptorrents.com" matched cert's "iptorrents.com"
*  issuer: C=GB; ST=Greater Manchester; L=Salford; O=COMODO CA Limited; CN=COMODO ECC Domain Validation Secure Server CA 2
*  SSL certificate verify ok.
> GET /login.php HTTP/1.1
> Host: iptorrents.com
> User-Agent: curl/7.54.1
> Accept: */*
> 
< HTTP/1.1 200 OK
< Date: Wed, 20 Sep 2017 00:21:09 GMT
< Content-Type: text/html; charset=UTF-8
< Transfer-Encoding: chunked
< Connection: keep-alive
< Set-Cookie: __cfduid=dd008e512f484825eb485a1f9821c4e4d1505866869; expires=Thu, 20-Sep-18 00:21:09 GMT; path=/; domain=.iptorrents.com; HttpOnly
< Cache-Control: no-store, no-cache, must-revalidate, max-age=1, post-check=0, pre-check=0, max-age=0
< Pragma: no-cache
< X-XSS-Protection: 1; mode=block
< X-Content-Type-Options: nosniff
< Referrer-Policy: same-origin
< X-Frame-Options: SAMEORIGIN
< Expires: Wed, 20 Sep 2017 00:21:09 GMT
< Server: cloudflare-nginx
< CF-RAY: 3a109f7d9d2e0f75-FRA
< 
<!DOCTYPE html>
<html style="height:100%">
        <head>
                <title>IPTorrents</title>
                <script src="/cdn-cgi/apps/head/Rpp7yz_yx1kVXUOgVdmvqTSiJJA.js"></script><link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
                <link rel="stylesheet" href="css/login.css" type="text/css" />
<script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
<script type="text/javascript" src='https://www.google.com/recaptcha/api.js?hl=en'></script>
        </head>
        <body style="height:100%;">
        <div style="width:100%;height:100%;">
                <div id="loginWrap" style="width:100%;padding-top:10px">
                        <div align="center">
                                        <img style="margin-right:-70px" src="/pic/iPT.png" />
                                </div>

                                <div class="flat-form" style="margin-top:10px">
                            <div id="login" class="form-action show">
<div align="center"><a href="/signup.php" class="tabLabel">Sign Up</a>  &laquo; &raquo; <a href="/recover.php" class="tabLabel">Forgot your password?</a></div>
<hr />



                                <form action="take_login.php" method="post">
                                    <ul>
                                        <li>
                                                <img src="/pic/login_username.svg" class="loginImg"/>
                                                <input type="text" name="username" class="username" autofocus placeholder="Email" required onfocus="this.style.boxShadow='0 0 7px #ffffbe'" onblur="this.style.boxShadow='none'" />
                                        </li>
                                        <li>
                                                <img src="/pic/login_password.svg" class="loginImg"/>
                                            <input type="password" class="password" name="password" placeholder="Password" required onfocus="this.style.boxShadow='0 0 7px #ffffbe'" onblur="this.style.boxShadow='none'"/>
                                        </li>
                                                                                                <li>
<input type="submit" value="Sign In" class="button" /><br>
                                        </li>
                                    </ul>
                                </form>
<hr /><a style="text-decoration: none" href="https://chat.iptorrents.com/?nick=HelpMe911"><div align="center" style="color:yellow; font-size:13px">If your account was disabled or you simply can't log in,<br>Click here to join our live irc #support channel.</div></a><hr />
<div align="center" style="font-size:13px">If you need help please contact us at: ipthelp@protonmail.com</div><br>
                            </div>
                        </div>
                </div>
                </div>
        </body>
* Connection #0 to host iptorrents.com left intact

**Comment by ryanmcgr** [comment]
https://pastebin.com/6zmKN7di

**Comment by kaso17** [comment]
Please update to v0.8.216 and retry + repost the log (should contain more details)

**Comment by ryanmcgr** [comment]
I upgraded and still no luck. Full logs are here - https://pastebin.com/TyA8jKBC

Thanks,
R

**Comment by ryanmcgr** [comment]
Fixed it with 
`wget https://curl.haxx.se/ca/cacert.pem`
`cert-sync --user cacert.pem`

Thanks,
